zerver/models.py

```diff
@@ -2106,6 +2123,8 @@ def email_address_is_realm_public(self) -> bool:
         return False
 
     def has_permission(self, policy_name: str) -> bool:
+        from zerver.lib.user_groups import is_user_in_group
+
```

Comment:
I was getting a circular dependency error if it was imported at the beginning of module.

Reply:
This is probably fine but deserves a comment. Later we might refactor this function away from living in `models.py` to some `zerver/lib/` space of its own to resolve that.